.github/workflows/dependent-issues.yml

```diff
@@ -0,0 +1,36 @@
+---
+name: Dependent Issues
+
+on:
+  issues:
+    types:
+      - opened
+      - reopened
+  pull_request_target:
+    types:
+      - opened
+      - reopened
+  pull_request:
+    types:
+      - opened
+      - reopened
+  schedule:
+    - cron: '0 * * * *' # check hourly
+
+jobs:
+  check:
+    name: check-dependent-PRs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: z0al/dependent-issues@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          # The label to use to mark dependent issues
+          label: dependent
+
+          # Enable checking for dependencies in issues.
+          check_issues: on
+
+          # A comma-separated list of keywords to mark dependency.
+          keywords: depends on, Depends on
```

Comment:
Will this work for something like:

> Depends on: <PR>

And will this not trigger for something like:

> Does this depend on <Issue>?

Reply:
yes. When you comment

```Depends on: <PR>```
or
```depends on: <PR>```